lib/recaptcha/adapters/controller_methods.rb

```diff
@@ -24,7 +24,8 @@ def verify_recaptcha(options = {})
               options = options.merge(remote_ip: remoteip.to_s) if remoteip
             end
 
-            Recaptcha.verify_via_api_call(recaptcha_response, options)
+            success, @_recaptcha_reply = Recaptcha.verify_via_api_call(recaptcha_response, options)
```

Comment:
this will have to pass in `with_reply: true` or something like that

Reply:
ah nvm ... `a, b = true` works out fine ... so the user can decide if they want the reply